### PR TITLE
Feat delete orphaned projects

### PR DESCRIPTION
**Pull request by shimonewman**

### CODE REVIEW

File: src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            ->inject('register')
+            ->callback(function (Group $pools, Cache $cache, Database $dbForConsole, Registry $register) {
+                $this->action($pools, $cache, $dbForConsole, $register);
+            });
+    }
+
+
+    public function action(Group $pools, Cache $cache, Database $dbForConsole, Registry $register): void
+    {
+        //docker compose exec -t appwrite delete-orphaned-projects
```
Comment (TorstenDittmann) [inline src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php:41]: leftover

File: src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    $message = ' (' . $collectionsCreated . ') collections where found on project (' . $project->getId() . '))';
+                    if ($collectionsCreated < ($totalProjects + 2)) {
+                        Console::error($message);
+                        $orphans++;
+                    } else {
+                        Console::log($message);
+                    }
+                } catch (\Throwable $th) {
+                    //$dbForConsole->deleteDocument('projects', $project->getId());
+                    //Console::success('Deleting  project (' . $project->getId() . ')');
```
Comment (TorstenDittmann) [inline src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php:93]: whats this doing?

**Review by TorstenDittmann — COMMENTED** [review]
I dont see any deletion going on 👍🏻

File: src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php (hunk 2 above)
Comment (shimonewman) [reply src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php:93]: This is supposed to do the actual deletion after the "dry run" is approved

File: src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php (hunk 1 above)
Comment (shimonewman) [reply src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php]: Yeah, removed

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**